src/ios/DoGoodGetGood.xcodeproj/project.pbxproj

```diff
@@ -744,8 +744,11 @@
 				PRODUCT_NAME = DoGoodGetGood;
 				PROVISIONING_PROFILE_SPECIFIER = "";
 				RELEASE_NUMBER = 1.1.2;
+				SUPPORTED_PLATFORMS = "iphoneos iphonesimulator";
+				SUPPORTS_MACCATALYST = NO;
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
 				SWIFT_VERSION = 5.0;
+				TARGETED_DEVICE_FAMILY = "1,iPhone SE (3rd generation)";
```

Comment:
This seems wrong? "1,iPhone SE (3rd generation)"
Is the 1 really correct there?

I'm quite suprised these line have been added...

Reply:
I think it's happened during the time of environment setup.We will fix it.

Reply:
<img width="1267" alt="Знімок екрана 2024-02-28 о 08 20 14" src="https://github.com/Do-Good-Get-Good/do-good-get-good/assets/71877080/d09ff0b9-7bb4-4e0d-be05-934ba818ebf3">


TARGETED_DEVICE_FAMILY = 1 means it can run only on iPhone
TARGETED_DEVICE_FAMILY = 2 means it can run only on iPad
TARGETED_DEVICE_FAMILY = 1,2 means it can run on both

I changed it from 1,2 to 1 because we don't want that this app runs on iPad, it's not adapted to iPad